src/neo/Network/P2P/Payloads/ExtensiblePayload.cs

```diff
@@ -0,0 +1,89 @@
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Persistence;
+using Neo.SmartContract;
+using System;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class ExtensiblePayload : IInventory
```

Comment:
Add `ValidIndexStart` and `ValidIndexEnd`? If the current block index is out of range, we can just drop the message.

Reply:
Maybe this depend to the plugin

Reply:
I think all `ExtensiblePayload` should have these fields.

Reply:
Done